.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Report a bug in the client library API
+title: ""
+labels: ""
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Reproduction**
+Include the code that is causing the error:
+
+```python
+# code here
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen. If applicable, add screenshots to help explain your problem.
+
+**Versions (please complete the following information):**
+
+<!--
+For EdgeDB version: Run `edgedb` from your project directory copying the first line of output or query `select sys::get_version_as_str();`
+For EdgeDB CLI version: Run `edgedb --version` from anywhere
+For `edgedb-python` version: Run `pip freeze` from your project directory and look for `edgedb`.
+For Python version: Run `python --version`
+-->
+
+- OS:
```

Comment:
Have you considered using GitHub's new issue template `.yaml` structure for adding checkboxes and required fields?